### ISSUE TITLE
fix breakage in mvfstandalone_tests

### ISSUE THREAD

**Issue by ftrader** [BTCfork/hardfork_prototype_1_mvf-bu#60]
see https://travis-ci.org/BTCfork/hardfork_prototype_1_mvf-bu/builds/187072114

**Comment by ftrader** [comment]
The general error pattern is
`test/mvfstandalone_tests.cpp(95):
error: in "mvfstandalone_tests/mvfgetconfigfile":
check MVFGetConfigFile() == GetDataDir() / BTCFORK_CONF_FILENAME has failed ["btcfork.conf" != "/some/temp/path/to/btcfork.conf"]`

**Comment by ftrader** [comment]
The issue was primarily caused by calling GetDataDir(false) in MVFGetConfigFile, but with default "true" parameter value in the unit test.

This was due to a wrong assumption I made about the function of the boolean parameter - I thought it constructed the directory if non-existent, but it regulates whether the returned datadir is specific to the network (which we actually want).

After using GetDataDir() in MVFGetConfigFile our btcfork.conf is now written to the network-specific datadir. Some MVF qa tests also had to be adapted.
